crates/uv-publish/src/lib.rs

```diff
@@ -704,12 +711,31 @@ pub async fn check_url(
 /// Calculate the requested hashes of a file.
 async fn hash_file(
     path: impl AsRef<Path>,
+    name: &str,
     hashers: Vec<Hasher>,
+    reporter: Option<&dyn Reporter>,
```

Comment:
This should use a reporter unconditionally, since all production code has a reporter.